book/routing.rst

```diff
@@ -1547,6 +1547,8 @@ a template helper function:
     The ``path()`` PHP templating helper was introduced in Symfony 2.8. Prior
     to 2.8, you had to use the ``generate()`` helper method.
 
+The ``path()`` method generates relative URLs.
```

Comment:
I think we should "blend" this into the paragraph before the code block, instead of a lost sentence somewhere at the end of the section. What about changing the paragraph to:

```diff
  The most common place to generate a URL is from within a template when linking
  between pages in your application. This is done just as before, but using
- a template helper function:
+ the ``path()`` function to generate a relative URI:
```